integr8ly/integreatly-operator
# INTLY-2667 codeready github integration

currently, a github identity provider is set up in the operator.
however codeready does not currently work with this github provider.

there was also an issue where the codeready or che client was being
removed from sso periodically. this also resolves that issue.

verification:
- have an existing openshift 4 cluster
- create an operator source, pointing at the registry `aidenkeating`
- create an installation custom resource, per the README in this repo
- wait for codeready to be provisioned
- ensure you update the github oauth app redirect uri to be the one
   from the github identity provider in rhsso
- login to codeready
- go to create a workspace and ensure you can add a project from
  github
- if the github project list loads, everything is good
